src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -16,8 +20,8 @@ public User login(String login, String password) throws AuthenticationException
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
         return user;
     }
 }
+
```

Comment:
Don't forget about empty lines at the end of files. There should be no empty lines after the last line of code.